tests/Support/Queries/PostQuery.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rebing\GraphQL\Tests\Support\Queries;
+
+use GraphQL\Type\Definition\Type;
+use Rebing\GraphQL\Support\Query;
+use Rebing\GraphQL\Support\SelectFields;
+use Rebing\GraphQL\Support\Facades\GraphQL;
+use Rebing\GraphQL\Tests\Support\Models\Post;
+
+class PostQuery extends Query
+{
+    protected $attributes = [
+        'name' => 'post',
+    ];
+
+    public function type()
+    {
+        return GraphQL::type('Post');
```

Comment:
Shouldn't this (and the other Queries as well) return `Type::nonNull(GraphQL::type('Post'))` instead?

Reply:
Ok nvm, I just saw the issue you posted. It's strange though, because I thought I've used nonNull with custom types